coderd/notifications/testdata/rendered-templates/smtp/TemplateUserAccountCreated.html.golden

```diff
@@ -39,8 +39,12 @@ l', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif; color: #020617=
 r: 1px solid #e2e8f0; border-radius: 8px; background-color: #fff; text-alig=
 n: left; font-size: 14px; line-height: 1.5;">
       <div style=3D"text-align: center;">
-        <img src=3D"https://coder.com/coder-logo-horizontal.png" alt=3D"Cod=
-er Logo" style=3D"height: 40px;" />
+        <img src=3D"https://coder.com/coder-logo-horizontal.png" alt=3D"Log=

```

Comment:
Looks like we're missing tests verifying this change. Can you check if it is easy to pick one SMTP template and render it using a custom logo/app name? It would be cool to have at least one golden file.